zipline/pipeline/expression.py

```diff
@@ -328,7 +326,9 @@ def __repr__(self):
 
     def short_repr(self):
         """Short repr to use when rendering Pipeline graphs."""
-        return "Expression: {expr}".format(
-            typename=type(self).__name__,
-            expr=self._expr,
+        final = re.sub(r"[-+]?\d*\.\d+",
```

Comment:
Can we add a comment explaining what's happening here?

Reply:
Yep